_security/access-control/users-roles.md

```diff
@@ -139,37 +139,36 @@ See [Create role mapping]({{site.url}}{{site.baseurl}}/security/access-control/a
+| `opensearch_dashboards_user`                     | Grants basic user access to OpenSearch Dashboards.                                                                                                                                                                                                                                                                                          |
+| `security_manager`                               | Grants permissions to manage security-related features and configurations.                                                                                                                                                                                                                                                                  |
+| `snapshot_management_full_access`                | Grants full permissions to all snapshot management actions.                                                                                                                                                                                                                                                                                 |
+| `snapshot_management_read_access`                | Grants permissions to view snapshot management actions and configurations, but not to modify them.                                                                                                                                                                                                                                          |
+| `point_in_time_full_access`                      | Grants full permissions to all Point in Time operations.                                                                                                                                                                                                                                                                                    |
+| `security_analytics_full_access`                 | Grants full permissions to all Security Analytics functionality.                                                                                                                                                                                                                                                                            |
+| `security_analytics_read_access`                 | Grants permissions to view the various components in Security Analytics, such as detectors, alerts, and findings. It also includes permissions that allow users to search for detectors and rules. This role does not allow a user to perform actions such as modifying or deleting a detector.                                             |
```

Comment:
Is this line formatting different?

Reply:
My editor formatted it on save but it made it easier to read on my end so I thought it was okay, also when the screen is wide enough (as wide as the widest line) the table break symbol doesn't wrap to the next line like that. 
So it's all as expected, it compiles correctly and previews correctly and is just formatted and when looking at the code, for me it reads better. I can also remove the whitespace if you prefer.